lib/event_tracer/datadog_logger.rb

```diff
@@ -18,15 +18,17 @@ class DatadogLogger < BasicDecorator
       gauge: :gauge,
       set: :set,
       histogram: :histogram
-    }
+    }.freeze
     DEFAULT_METRIC_TYPE = :count
     DEFAULT_COUNTER = 1
+    DEFAULT_TAGS = [:environment].freeze # These should be appended by the Logger itself
```

Comment:
This doesn't allow configuration per application. Each application can have a different set of default tags, so we shouldn't decide for them.